src/werkzeug/datastructures.py

```diff
@@ -355,6 +355,15 @@ def __setstate__(self, value):
         dict.clear(self)
         dict.update(self, value)
 
+    def __iter__(self):
+        """This looks like a no-op, but actually preserves required interop.
+
+        See https://github.com/pallets/werkzeug/pull/2043.
+        """
+        return dict.__iter__(self)
+        # return super().__iter__() also works here, which makes this look
+        # even more like it should be a no-op, yet actually changes behavior.
```

Comment:
Am I the only one who finds a comment after the last statement in a block a bit strange?